Content.Server/Speech/EntitySystems/SouthernAccentSystem.cs

```diff
@@ -21,8 +21,15 @@ private void OnAccent(EntityUid uid, SouthernAccentComponent component, AccentGe
 
         //They shoulda started runnin' an' hidin' from me!
         message = Regex.Replace(message, @"ing\b", "in'");
+        message = Regex.Replace(message, @"ING\b", "IN'");
+
         message = Regex.Replace(message, @"\band\b", "an'");
+        message = Regex.Replace(message, @"\bAnd\b", "An'");
+        message = Regex.Replace(message, @"\bAND\b", "AN'");
+
         message = Regex.Replace(message, "d've", "da");
+        message = Regex.Replace(message, "D'VE", "DA");
+
         args.Message = message;
     }
```

Comment:
One of the other accents keeps casing, look at that rather than baking these in.